.github/workflows/openshift-nightly.yml

```diff
@@ -0,0 +1,31 @@
+name: OpenShift Nightly Job
+on: 
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    name: OpenShift Nightly Job
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout v2
+        uses: actions/checkout@v2
+      - name: Authenticate and set context
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
```

Comment:
do the secrets have to be updated ever?

Reply:
The token we use is a permanent token, so it should not need to be updated. @ksubrmnn

Reply:
Yep that's correct! The server and token should stay the same